### PR TITLE
EQV Fix

### PR DESCRIPTION
**Pull request by vkbo**
.eqv. operator doesn't have the precedence parts of the code assumes.
Also removed a few that were a bit inconsistent.

### CODE REVIEW

**Comment by vkbo** [comment]
The precedence of .eqv. is indeed a little muddy.

http://jick.net/Manuals/FORTRAN/g77/Equivalence-Versus-Equality.html

File: source/mod_ffield_aux.f90 [hunk 1]
```diff
@@ -509,7 +509,7 @@ subroutine ReadVectPotCoeff(this,lun,norm,iErr)
           enddo
         endif
 
-        if (CoefSave.eqv..False.) then ! Create new coef
+        if (CoefSave.eqv..false.) then ! Create new coef
```
Comment (amereghe) [inline source/mod_ffield_aux.f90]: what about `if (.not.CoefSave)` ?

File: source/mod_ffield_aux.f90 [hunk 2]
```diff
@@ -531,7 +531,7 @@ subroutine ReadVectPotCoeff(this,lun,norm,iErr)
...
```
Comment (amereghe) [inline source/mod_ffield_aux.f90]: what about `if (.not.CoefSave)` ?

File: source/mod_ffield_aux.f90 [hunk 3]
```diff
@@ -553,7 +553,7 @@ subroutine ReadVectPotCoeff(this,lun,norm,iErr)
...
```
Comment (amereghe) [inline source/mod_ffield_aux.f90]: what about `if (.not.CoefSave)` ?

**Comment by vkbo** [comment]
> fine.
> not clear to me though if you wanted to purge `.eqv.` and `.neqv.` statements, though

I replaced the ones in mod_alloc because they were inconsistent, and I was the one who put them there. I left the ones in the fringe field code be because those modules are maintained by someone else.

.eqv. is technically the correct way to compare logicals though. But for simple if statements it's perhaps a bit overkill.

File: source/coll_db.f90 [hunk 4]
```diff
@@ -332,7 +332,7 @@ subroutine cdb_writeDB_ROOT
   character(len=5)          :: this_material = C_NULL_CHAR
   integer j
 
-  if(root_flag .eqv. .false. .or. root_CollimationDB /= 1) return
+  if((root_flag .eqv. .false.) .or. root_CollimationDB /= 1) return
```
[the remark this answers is not in the document]
Comment (vkbo) [reply source/coll_db.f90]: No, the one above is as it should be.

**Pull request merged**